src/seedsigner/models/settings_definition.py

```diff
@@ -335,6 +335,7 @@ def map_network_to_embit(cls, network) -> str:
     SETTING__BIP85_CHILD_SEEDS = "bip85_child_seeds"
     SETTING__ELECTRUM_SEEDS = "electrum_seeds"
     SETTING__MESSAGE_SIGNING = "message_signing"
+    SETTING_SHAMIR = "shamir_seeds"
```

Comment:
double underscore to separate category from named value.

Reply:
Also I would organize this directly below Electrum Seeds, both here in the constant declaration and below when the `SettingsEntry` is defined. This way all the seed type settings are grouped together.